scripts/run-integration-tests.travis.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 set -ev
 docker build -t one-app:at-test .
-if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then
+if [[ "${TRAVIS_BRANCH}" = "prerelease" ]] ||
```

Comment:
I think we should switch prerelease for release-candidate or prepare-release. prerelease could confuse

Reply:
> Do we have in place a way to make sure >= 1 One Amex Admins need to approve the release PR?

yes the admins would be added automatically